spec/activerecord-multi-tenant/model_extensions_spec.rb

```diff
@@ -132,6 +132,17 @@ def self.name
   end
 
   describe 'inspect method filters senstive column values' do
+    if ActiveRecord::VERSION::MAJOR >= 7 && ActiveRecord::VERSION::MINOR >= 2
+      # related: https://github.com/rails/rails/pull/49765
+      around do |example|
+        prev = Account.attributes_for_inspect
+        Account.attributes_for_inspect = :all
+        example.run
+      ensure
+        Account.attributes_for_inspect = prev
+      end
```

Comment:
In Rails 7.2, `#inspect' doesn't print all attributes. This changes will output all attributes for compatibility.

related: https://github.com/rails/rails/pull/49765